sdcflows/transform.py

```diff
@@ -20,8 +20,35 @@
 #
 #     https://www.nipreps.org/community/licensing/
 #
-"""The :math:`B_0` unwarping transform formalism."""
+r"""
+The :math:`B_0` unwarping transform formalism.
+
+This module implements a data structure to represent the displacements field associated
+to the deformations caused by susceptibility-derived distortions.
+This implementation attempts to provide a single representation of such distortions independently
+of the estimation strategy (see :doc:`/methods`).
+
+.. _bspline-interpolation:
+
+That is achieved by implementing multi-level B-Spline cubic interpolators.
+For one given level, a function :math:`f(\mathbf{s})` can be represented as a linear combination
+of tensor-product cubic B-Spline basis (:math:`\Psi^3(\mathbf{k}, \mathbf{s})`;
+see Eq. :math:`\eqref{eq:2}`).
+
+
+.. math::
+
+    f(\mathbf{s}) =
+    \sum_{k_1} \sum_{k_2} \sum_{k_3} c(\mathbf{k}) \Psi^3(\mathbf{k}, \mathbf{s}).
+    \label{eq:1}\tag{1}
+
+
+"""
+import os
```

Comment:
We should take advantage of the free backport to get cleaner annotations, allowing us to get the behavior in the comments below
```suggestion
"""
from __future__ import annotations

import os
```

Reply:
Note that while we use `from __future__ import annotations`, we will not be able to use type annotations with `attrs` in this file. We do not currently, so this is fine.